packages/scrubber-bar/src/scrubber-bar.ts

```diff
@@ -0,0 +1,185 @@
+import {
+  LitElement,
+  html,
+  css,
+  customElement,
+  property,
+  PropertyValues,
+  TemplateResult,
+  CSSResult,
+} from 'lit-element';
+
+@customElement('scrubber-bar')
+export default class ScrubberBar extends LitElement {
+  @property({ type: Number }) percentComplete = 0;
+
+  userInteracting = false;
+
+  render(): TemplateResult {
+    return html`
+      <div class="container">
+        <input
+          id="slider"
+          type="range"
+          min="0"
+          max="100"
+          step="0.1"
+          value=${this.percentComplete}
+          @mousedown=${this.interactionStarted}
+          @mouseup=${this.interactionEnded}
+          @touchstart=${this.interactionStarted}
+          @touchend=${this.interactionEnded}
+          @input=${this.handleSlide}
+          @change=${this.handleSlide} />
+        <div id="webkit-range-input-style"></style>
```

Comment:
Question: does this `<div>` need a closing tag?

Reply:
Good catch!